packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -673,3 +743,12 @@ async function areFoldersEmpty(folderPaths: string[]): Promise<boolean> {
   }
   return isEmpty;
 }
+
+const FeatureFlagNewFolderStructure = "TEAMSFX_ARM_NEW_FOLDER_STRUCTURE";
```

Comment:
this seems removed by @qinezh , can you confirm with him?

Reply:
Thanks @blackchoey to mention. 

Yeah, this feature flag has already been removed and replaced with the existing feature flag `TEAMFX_MULTI_ENV`.

Reply:
It is removed, thanks.